From bb4f722685eca92d67bded58ac3ba81801a29c4e Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Fri, 19 Jul 2013 17:22:25 +0800
Subject: [PATCH 018/115] ENGR00215174 make fsl.imx jar as static library
 linked to wfd apk.

make fsl.imx jar as static library linked to wfd apk.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 WfdSink/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/WfdSink/Android.mk b/WfdSink/Android.mk
index 0de8dcd..1c65982 100644
--- a/WfdSink/Android.mk
+++ b/WfdSink/Android.mk
@@ -8,7 +8,7 @@ LOCAL_SRC_FILES := $(call all-java-files-under, src)
 LOCAL_PACKAGE_NAME := WfdSink
 LOCAL_CERTIFICATE := platform
 
-LOCAL_JAVA_LIBRARIES := fsl.imx
+LOCAL_STATIC_JAVA_LIBRARIES := fsl.imx
 
 include $(BUILD_PACKAGE)
 
-- 
1.8.0

